repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py

```diff
@@ -0,0 +1,50 @@
+import csv
+import os
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import RepositoriesMap, RepositoryMap
+from leapp.models.fields import ModelViolationError
+
+
+def inhibit_upgrade(title):
+    summary = 'Read documentation at: https://access.redhat.com/articles/3664871 for more information ' \
+        'about how to retrieve the files'
+    reporting.report_generic(title=title, summary=summary, severity='high', flags=['inhibitor'])
+    raise StopActorExecutionError(message=title, details={'details': summary})
+
+
+def scan_repositories(path):
+    if not os.path.isfile(path):
+        inhibit_upgrade('Repositories map file not found ({})'.format(path))
+
+    with open(path) as f:
+        if len(f.readlines()) == 0:
```

Comment:
If the idea is to check if file is not empty you could use "os.path.getsize(path)". That would be more descriptive of your intention and no need to call an "open"